include/vitals.inc.php

```diff
@@ -164,12 +169,14 @@ function my_null_slashes($string) {
 		$_SESSION['prefs']['PREF_THEME'] = get_default_theme();
 	}
 
-	$savant->addPath('template', AC_INCLUDE_PATH . '../themes/' . $_SESSION['prefs']['PREF_THEME'] . '/');
+	
+	// Add folders
```

Comment:
Please replace with a more meaningful code comment. This applies throughout the pull request.